compiler_opt/rl/compilation_runner.py

```diff
@@ -304,6 +332,8 @@ def __init__(self,
     self._compilation_timeout = (
         compilation_timeout or _COMPILATION_TIMEOUT.value)
     self._cancellation_manager = WorkerCancellationManager()
+    self._observers = [f() for f in create_observer_fns
```

Comment:
better indenting to improve the readability

Reply:
fixed.